docs/layer.md

```diff
@@ -25,53 +23,56 @@ which helps avoid id collisions in this case.
 
 E.g. assuming a composite GeoJsonLayer layer that renders two sublayers,
 choropleths and lines, with those ids:
-```
```

Comment:
If we remove the triple quotes, the examples will not be formatted as code. Is this intentional?